config/database.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+mongoose.connect(process.env.DATABASE_URL || 'mongodb://localhost/FitnessTracking', { 
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useUnifiedTopology: true 
+});
+
+const db = mongoose.connection;
+// database connection event
+db.on('connected', function () {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
db.on("connected", function () {
```